internal/collector/testdata/test-otelcol.yaml

```diff
@@ -0,0 +1,75 @@
+receivers:
```

Comment:
we store configs like these under `test/config/*` historically

Reply:
Moved it there 👍